requirements/edx/github.txt

```diff
@@ -76,6 +76,10 @@ git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc8685
 # This dependency will be removed by this work: https://openedx.atlassian.net/browse/PLAT-1660
 git+https://github.com/edx/django-rest-framework-oauth.git@0a43e8525f1e3048efe4bc70c03de308a277197c#egg=djangorestframework-oauth==1.1.1
 
+# Why a django-celery fork? To add Django 1.11 compatibility to the abandoned repo.
+# This dependency will be removed by the Celery 4 upgrade: https://openedx.atlassian.net/browse/PLAT-1684
+git+https://github.com/edx/django-celery.git@f87c6f914a1410463f54aebf68458c0653b20602#egg=django-celery==3.2.1+edx.1
```

Comment:
I guess it's time to start looking into running our own package server for these forks and such.  There was some talk of it about a year ago, but I haven't heard anything since.  These git dependencies get reinstalled every single time we check for updated dependencies, which isn't exactly helping our execution times.